### PR TITLE
translation for DE added

### PR DESCRIPTION
**Pull request by appleonkel**
### Checklist

*   [x ] I read [Contribution Guidelines](https://github.com/apragacz/django-rest-registration/blob/master/CONTRIBUTING.md#pull-requests)
*   [ ] I ran the checks and the tests (`make check && make test`) before submitting the PR on my branch and they passed
*   [ ] I attached unit tests testing the provided code so the code coverage will not drop below 98%

### Description
Added german translation file

### Why the change is needed?
Have one more translation

### CODE REVIEW

File: rest_registration/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+msgid "Invalid signature"
+msgstr "ungültige Signatur"
+
+#: exceptions.py:106
+msgid "Could not find verification templates"
+msgstr "Es konnten keine Verifizierungstemplate gefunden werden"
+
+#: utils/email.py:104
+msgid "No 'subject' key found"
+msgstr "Kein 'Betreff' gefunden"
```
Comment (apragacz) [inline rest_registration/locale/de/LC_MESSAGES/django.po:114]: In that case, we're asking about key explicitly named `'subject'` which is part of Django-REST-Registration email template configuration . My suggestion would be:
```suggestion
msgstr "Kein 'subject' Schlüssel gefunden"
```

Comment (appleonkel) [reply rest_registration/locale/de/LC_MESSAGES/django.po]: Thank you! I should have looked at the code.
It is the key of a dictionary.

**Comment by apragacz** [comment]
@appleonkel Thanks for your contribution!

**Comment by apragacz** [comment]
This change should be available in version v0.8.2+.

**Pull request merged**